exercises/practice/allergies/allergies_test.go

```diff
@@ -7,11 +7,13 @@ import (
 
 func TestAllergies(t *testing.T) {
 	for _, test := range listTests {
-		if actual := Allergies(test.score); !sameSliceElements(actual, test.expected) {
-			t.Fatalf("FAIL: Allergies(%d): expected %#v, actual %#v", test.score, test.expected, actual)
-		} else {
-			t.Logf("PASS: Allergic to %#v", test.expected)
-		}
+		t.Run(test.description, func(t *testing.T) {
+			if actual := Allergies(test.score); !sameSliceElements(actual, test.expected) {
+				t.Fatalf("FAIL: Allergies(%d): expected %#v, actual %#v", test.score, test.expected, actual)
+			} else {
+				t.Logf("PASS: Allergic to %#v", test.expected)
```

Comment:
With subtests, a string logging a `PASS` or `FAIL` is redundant and can be removed as running `go test -v` will make Go print explicitly the result for every test.

Reply:
Should be removed nearly everywhere, might have forgotten/not seen sometimes

Reply:
I'll make sure to double check that, no worries :)